includes/functions.php

```diff
@@ -173,6 +182,21 @@ function checkDiscovery($array, $sysObjectId, $sysDescr)
             if (preg_match_any($sysObjectId, $value) == $check) {
                 return false;
             }
+        } elseif ($key == 'snmpget') {
+            $options = isset($value['options']) ? $value['options'] : '-Oqv';
+            $mib = isset($value['mib']) ? $value['mib'] : null;
+            $mib_dir = isset($value['mibdir']) ? $value['mibdir'] : null;
+            $op = isset($value['op']) ? $value['op'] : 'contains';
+
+            $get_value = snmp_get($device, $value['oid'], $options, $mib, $mib_dir);
+
+            if (is_string($get_value)) {
+                $get_value = trim($get_value, '"');
```

Comment:
We really need to move this check into snmp_get() imho. Trimming " is littered everywhere within the code and I just can't see where doing it would cause us issues.